src/DecoderPlugin.php

```diff
@@ -39,11 +52,13 @@ public function __construct($useContentEncoding = true)
      */
     public function handleRequest(RequestInterface $request, callable $next, callable $first)
     {
-        $request = $request->withHeader('TE', ['gzip', 'deflate', 'compress', 'chunked']);
+        $encodings = extension_loaded('zlib') ? ['gzip', 'deflate', 'compress'] : [];
 
-        if ($this->useContentEncoding) {
-            $request = $request->withHeader('Accept-Encoding', ['gzip', 'deflate', 'compress']);
+        if ($this->useContentEncoding && count($encodings)) {
```

Comment:
is it correct that servers that respect Accept-Encoding will understand us when not sending this header? or should we send an empty header or something to say we can't handle encodings?